Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -0,0 +1,851 @@
+"""Automox Integration for Cortex XSOAR (aka Demisto)
+
+This integration empowers users of Cortex XSOAR with the
+capabilities provided by the Automox API.
+
+Organization, group, device, policy, and patch management all at your fingertips.
+
+For API reference, visit: https://developer.automox.com/
+"""
+
+import base64
+import string
+import traceback
+from typing import Any, Dict, List
+
+import demistomock as demisto
+import requests
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+''' CONSTANTS '''
+VERSION = "1.0.0"
+USER_AGENT = f'ax:PaloAltoNetworks-XSOAR-plugin/{VERSION}'
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'  # ISO8601 format with UTC, default in XSOAR
+
+# Common keys used in command arguments
+ORG_IDENTIFIER = 'org_id'
+DEVICE_IDENTIFIER = 'device_id'
+GROUP_IDENTIFIER = 'group_id'
+LIMIT_IDENTIFIER = 'limit'
+PAGE_IDENTIFIER = 'page'
+
+DEFAULT_ORG_ID = demisto.params().get(ORG_IDENTIFIER, None)
+
+''' CLIENT CLASS '''
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def _get_list_results(self, method: string, url_suffix: string, params: dict) -> List[Dict]:
+        results = []
+
+        result_limit = int(params['limit'])
+        page_limit = 250 if int(params['limit']) > 250 else int(params['limit'])
+
+        params['limit'] = page_limit
+
+        while result_limit > 0:
+            response = self._http_request(
+                method=method,
+                url_suffix=url_suffix,
+                params=params
+            )
+
+            # This is necessary for vuln sync, since those responses have metadata & data keys.
+            if isinstance(response, dict) and isinstance(response['data'], list):
+                response = response['data']
+
+            results += response
+
+            if len(response) < page_limit or len(response) == 0:
+                break
+
+            result_limit -= len(response)
+
+        return results
+
+    def action_on_vulnerability_sync_batch(self, org_id: int, batch_id: int, action: string):
+        url_suffix = f"/orgs/{org_id}/tasks/batches/{batch_id}"
+
+        if action == "accept":
```

Comment:
Could do `if action in ("accept", "reject"):` and then append the action to url_suffix on line 89